jsonrpc/jsonrpc.go

```diff
@@ -32,11 +33,16 @@ func (s serverType) String() string {
 	}
 }
 
+type Handlers struct {
```

Comment:
Instead of creating a new struct, add this to the Config field, it is meant to add these type of configurations.